src/test/java/juja/microservices/gamification/slackbot/dao/RestUserRepositoryTest.java

```diff
@@ -68,7 +68,7 @@ public void shouldReturnUserWhenSendUserDataToRemoteUserService() {
     @Test
     public void shouldReturnUuidUserWhenSendUserDataToRemoteUserService() {
         //given
-        mockServer.expect(requestTo("/users/search/slackNickname=@user"))
+        mockServer.expect(requestTo("http://user.juja.com.ua/users/search/slackNickname=@user"))

```

Comment:
read URLs from properties, like in the RestUserRepository.
because if we will change the URL into property file, the test will crash.